Microsoft.Health.sln

```diff
@@ -96,6 +96,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Encryption
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Encryption.UnitTests", "src\Microsoft.Health.Encryption.UnitTests\Microsoft.Health.Encryption.UnitTests.csproj", "{E2D4F975-9F71-416B-96E5-F2A5101FCBCD}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "lang", "lang", "{99CA0B4C-C0EB-44B7-9C06-F0C8725D9A6C}"
```

Comment:
This source file was already present, but I thought I'd add it to the solution or visibility